src/http/routing.rs

```diff
@@ -2979,6 +2982,13 @@ impl<'a> RouteInfo<'a> {
                 Route::ChannelsIdPinsMessageId(channel_id),
                 Cow::from(Route::channel_pin(channel_id, message_id)),
             ),
+            RouteInfo::CreateForumThread {
+                channel_id
+            } => (
+                LightMethod::Post,
+                Route::ChannelsIdThreads(channel_id),
+                Cow::from(Route::channel_private_threads(channel_id)),
```

Comment:
While this works, the [API specs](https://discord.com/developers/docs/resources/channel#start-thread-in-forum-channel) have a separate definition for creating threads in forums. The type of the created thread is PUBLIC_THREAD. Both routes are identical, which is why the current implementation works, but it is better to stick to the specification and create a separate route for creating threads in forums as they have distinct differences to creating a regular, private thread, which the current implementation implies. This route also has different JSON parameters, though this is also not considered in the route of creating a private thread.